library/step.go

```diff
@@ -34,6 +35,27 @@ type Step struct {
 	Distribution *string `json:"distribution,omitempty"`
 }
 
+// Duration calculates and returns the total amount of
+// time the step ran for in a human-readable format.
+func (s *Step) Duration() string {
+	// check if the service doesn't have a started or finished timestamp
```

Comment:
Change to `check if the step doesn't...`

Reply:
This should be addressed